source/client/index.js

```diff
@@ -26,6 +26,7 @@ module.exports = ({
   routes = ensureRoutes('createClientApp'),
   basepath = '',
   history = createHistory(),
+  shouldUpdateScroll = defaultUpdateScroll,
```

Comment:
Never mind me: `shouldUpdateScroll` is in the new scroll behaviour docs. Good move!